_posts/2017-11-26-stammtisch-17.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Stammtisch in 2017
+date:   2017-11-26
+categories: story
+---
+This year we decided to try out a new format. Every third Wednesday of the month we would have our monthly meet up, or Stammtisch (the German word for regular’s table).
+
+We started our journey in August, sitting around a long table by the Spree, and it has been an amazing open format. There are no presentations here, only community members networking, discussing shared interests or asking for help.
+
+We’ve talked about ways we can encourage more women to study engineering , how to improve internships and maybe reach out to the companies we work for to give them our community feedback. We have met like minded women and men who have shared their stories with us and helped us grow as a community. We’ve formed new friendships, relationships and all in all increased awareness of successful and struggle stories.
+
+As the year comes to an end, we are delighted to see familiar faces from our monthly meet ups at our events. It’s incredible to see people connecting and supporting each other for the better. I am personally excited to see our community members getting more involved and offering their time and help. 
+
+We have grown to 1070 members here at Women Techmakers Berlin and I look forward to continuing this journey in 2018.
```

Comment:
@corinagh 1070 is on the meetup website alone! Can you add that? Or alternatively add the members from other platform?

Reply:
I wouldn't add the members from other platforms, because there's probably a pretty large overlap :)

Reply:
It's not my first choice either, but I did learn it's a common practice to offered the option.